solutions/swb-reference/src/SWBStack.ts

```diff
@@ -274,13 +275,49 @@ export class SWBStack extends Stack {
       createAccountHandler
     );
 
+    // RevokedTokens DynamoDB Encryption Key
+    const revokedTokensDDBTableEncryptionKey: WorkbenchEncryptionKeyWithRotation =
+      new WorkbenchEncryptionKeyWithRotation(this, `${this.stackName}-revokedTokensDDBTableEncryptionKey`);
+
+    // Create RevokedTokens DynamoDB Table
+    const revokedTokensDDBTable = this._createRevokedTokensDDBTable(
+      revokedTokensDDBTableEncryptionKey.key,
+      apiLambda,
+      statusHandler,
+      createAccountHandler
+    );
+
     // Add DynamicAuth DynamoDB Table name to lambda environment variable
     _.map([apiLambda, statusHandler, createAccountHandler], (lambda) => {
       lambda.addEnvironment('DYNAMIC_AUTH_DDB_TABLE_NAME', dynamicAuthTable.tableName);
     });
 
+    // Add RevokedTokens DynamoDB table name to lambda environment variable
+    _.map([apiLambda, statusHandler, createAccountHandler], (lambda) => {
+      lambda.addEnvironment('REVOKED_TOKENS_DDB_TABLE_NAME', revokedTokensDDBTable.tableName);
+    });
+
     const apiGwUrl = this._createRestApi(apiLambda);
 
+    [apiLambda, statusHandler, createAccountHandler].forEach((lambda) => {
+      if (
+        !_.isUndefined(lambda.node.findChild('ServiceRole').node.tryFindChild('DefaultPolicy')) &&
+        !_.isUndefined(lambda.node.findChild('ServiceRole').node.findChild('DefaultPolicy').node.defaultChild)
+      ) {
+        const cfnPolicy = lambda.node.findChild('ServiceRole').node.findChild('DefaultPolicy').node
+          .defaultChild as CfnPolicy;
+        cfnPolicy.addMetadata('cfn_nag', {
+          // eslint-disable-next-line @typescript-eslint/naming-convention
+          rules_to_suppress: [
+            {
+              id: 'W76',
+              reason: 'TODO: SPCM for IAM policy document is higher than 25'
```

Comment:
We want to avoid TODOs; can you update the reason as to why we are suppressing this? I thought the default for cfn nag SPCM rule is 50?